Modules/MUON/MCH/include/MCH/PedestalsTask.h

```diff
@@ -36,30 +35,28 @@ class PedestalsTask final : public TaskInterface
   void initialize(o2::framework::InitContext& ctx) override;
   void startOfActivity(Activity& activity) override;
   void startOfCycle() override;
-  void monitorDataReadout(o2::framework::ProcessingContext& ctx);
-  void monitorDataDigits(const o2::framework::DataRef& input);
   void monitorData(o2::framework::ProcessingContext& ctx) override;
   void endOfCycle() override;
   void endOfActivity(Activity& activity) override;
   void reset() override;
 
  private:
-  Decoder mDecoder;
-  uint64_t nhits[MCH_MAX_CRU_IN_FLP][24][40][64];
-  double pedestal[MCH_MAX_CRU_IN_FLP][24][40][64];
-  double noise[MCH_MAX_CRU_IN_FLP][24][40][64];
+  o2::mch::raw::Solar2FeeLinkMapper mSolar2FeeLinkMapper;
+  o2::mch::raw::Elec2DetMapper mElec2DetMapper;
+
+  /// helper class that performs the actual computation of the pedestals from the input digits
+  o2::mch::calibration::PedestalProcessor mPedestalProcessor;
 
   //Matrices [de][padid], stated an upper value for de# and padid#
 
-  uint64_t nhitsDigits[1100][1500];
-  double pedestalDigits[1100][1500];
-  double noiseDigits[1100][1500];
+  //uint64_t nhitsDigits[1100][1500];
```

Comment:
I guess the rules for QC are the same as for AliceO2 and so commented code should be removed simply ?

Reply:
done